Modules/IO/NIFTI/wrapping/itkNiftiImageIO.wrap

```diff
@@ -5,3 +5,4 @@ itk_wrap_include("itkNiftiImageIOFactory.h")
 itk_wrap_simple_class("itk::NiftiImageIO" POINTER)
 itk_wrap_simple_class("itk::NiftiImageIOFactory" POINTER)
 itk_wrap_simple_class("itk::Analyze75Flavor" ENUM)
+itk_wrap_simple_class("itk::NiftiFileEnum")
```

Comment:
Since `itk::NiftiImageIO` uses `itk::NiftiFileEnum`, we want the wrapping of `itk::NiftiFileEnum` to be declared before `itk::NiftiImageIO`.

Reply:
The same should apply to `itk::Analyze75Flavor` then, right? If so, I'll do that in a separate PR.

Having missed the `ENUM` specifier is causing the failure I think.

Reply:
> The same should apply to itk::Analyze75Flavor then, right? If so, I'll do that in a separate PR.

Makes sense.

> Having missed the ENUM specifier is causing the failure I think.

The `ENUM` is no longer required following the strongly typed upgrade.

Reply:
> Since itk::NiftiImageIO uses itk::NiftiFileEnum, we want the wrapping of itk::NiftiFileEnum to be declared before itk::NiftiImageIO.

Done in b1d159b.

> The same should apply to itk::Analyze75Flavor then, right? If so, I'll do that in a separate PR.
> Makes sense.

Done in PR #2318.